scripts/prepare_alpaca.py

```diff
@@ -18,7 +18,7 @@
 DATA_FILE_NAME = "alpaca_data_cleaned_archive.json"
 DESTINATION_PATH = Path("data/alpaca")
 CHECKPOINT_DIR = Path("checkpoints/stabilityai/stablelm-base-alpha-3b")
-TEST_SPLIT_SIZE = 2000
+TEST_SPLIT_PERCENT = 10
```

Comment:
Can you check how many samples does alpaca-lora have in total? We should set the percentage that corresponds to a test split size of 2k to keep the same behaviour

Reply:
Sure.  There are 51759 samples, so 4% would give us 2070 in the test set.

Reply:
Just a side comment: seems like 2K samples out of 52K is rather small for the test set, no?  I'm just starting to work on training LLMs, but for image detection/segmentation, 10-20% seems to be the sweet spot.

Reply:
Done.  test_split_percent is now a float, and the default produces exactly 2000 test samples.